### PR TITLE
doc(ingestion): looker & lookml ingestion guide

### PR DESCRIPTION
**Pull request by siddiquebagwan**
static assets PR: https://github.com/datahub-project/static-assets/pull/37

### CODE REVIEW

**Comment by hsheth2** [comment]
@mohdsiddique looks like the docs site build is failing with a lint error

**Comment by mohdsiddique** [comment]
> @mohdsiddique looks like the docs site build is failing with a lint error

It is fixed now

**Comment by hsheth2** [comment]
@yoonhyejin could you take a pass over this for formatting, readability, clarity, and writing style?

**Comment by yoonhyejin** [comment]
@mohdsiddique sorry i was unaware of this! seems like the path for the images are broken
image

File: docs/quick-ingestion-guides/looker/configuration.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+1.  Navigate to the **Sources** tab and click **Create new source**
+
+  <p align="center">
+    <img width="75%" alt="Click &quot;Create new source&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_click_create_new_source_button.png"/>
+  </p>
+
+2.  Choose Looker
+
+  <p align="center">
+    <img width="70%" alt="Select Looker from the options" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-choose-looker.png"/>
```
Comment (yoonhyejin) [inline docs/quick-ingestion-guides/looker/configuration.md:62]: the image is broken

File: docs/quick-ingestion-guides/looker/overview.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## What You Will Get Out of This Guide
+
+This guide will help you set up the Looker & LookML connectors to begin ingesting metadata into DataHub.
+ 
+
+Upon completing this guide, you will have a recurring ingestion pipeline that will extract metadata from Looker & LookML and load it into DataHub. This will include to following Looker asset types:
+
+**Looker Connector Ingested Assets:**
+
+* Dashboards
```
Comment (yoonhyejin) [inline docs/quick-ingestion-guides/looker/overview.md:15]: I think the format here could be improved

image

Comment (yoonhyejin) [reply docs/quick-ingestion-guides/looker/overview.md:15]: This should be more look like this (please refer to the [bigquery quickstart guide](https://datahubproject.io/docs/quick-ingestion-guides/bigquery/setup/#bigquery-prerequisites) : 
image

File: docs/quick-ingestion-guides/looker/overview.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* Charts
+* Explores 
+* Schemas 
+* Owners of Dashboards
+
+    To get complete Looker metadata integration (including Looker views and lineage to the underlying warehouse tables), you must also use the `lookml` connector.
```
Comment (yoonhyejin) [inline docs/quick-ingestion-guides/looker/overview.md:21]:
```suggestion
:::note
To get complete Looker metadata integration (including Looker views and lineage to the underlying warehouse tables), you must also use the `lookml` connector.
:::
```

File: docs/quick-ingestion-guides/looker/overview.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+
+**LookML Connector Ingested Assets**
+
+* LookML views from model files in a project
+* Metadata for dimensions
+* Metadata for measures
+* Dimension Groups as tag
+
+    To get complete Looker metadata integration (including Looker dashboards and charts and lineage to the underlying Looker  views, you must also use the `looker` connector.
```
Comment (yoonhyejin) [inline docs/quick-ingestion-guides/looker/overview.md:33]:
```suggestion
:::note
To get complete Looker metadata integration (including Looker dashboards and charts and lineage to the underlying Looker  views, you must also use the `looker` connector.
:::
```

File: docs/quick-ingestion-guides/looker/setup.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+   a. Go to `New Permission Set`
+
+      <p align="center">
+      <img width="75%" alt="Looker new permission set" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-new-permission-set-button.png"/>
+      </p>
+
+   b. Set name for  `New Permission Set`, says `DataHub Connector Permission Set` and select following permissions
+
+      - access_data
```
Comment (yoonhyejin) [inline docs/quick-ingestion-guides/looker/setup.md:36]: also here, the line spacing should be narrower 
or you could utilize <details></details> to make it collapsible 

image

File: docs/quick-ingestion-guides/looker/configuration.md [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+2. Ensure you've configured your correct timezone
+<p align="center">
+    <img width="75%" alt="timezone_selector" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_set_execution_timezone.png"/>
+</p>  
+
+3. Click **Next** when you are done
+
+## Finish Up
+
+1. Name your ingestion source, then click **Save and Run**
```
Comment (yoonhyejin) [inline docs/quick-ingestion-guides/looker/configuration.md:115]:
```suggestion
Name your ingestion source, then click **Save and Run**
```

I don't think we need numbering here 
image

**Review by yoonhyejin — CHANGES_REQUESTED** [review]

File: docs/quick-ingestion-guides/looker/configuration.md (hunk 1 above)
Comment (mohdsiddique) [reply docs/quick-ingestion-guides/looker/configuration.md:62]: @yoonhyejin 
We need to merge this static PR first: https://github.com/datahub-project/static-assets/pull/37

Comment (hsheth2) [reply docs/quick-ingestion-guides/looker/configuration.md:62]: merged that PR

File: docs/quick-ingestion-guides/looker/overview.md (hunk 3 above)
Comment (siddiquebagwan) [reply docs/quick-ingestion-guides/looker/overview.md:21]: done

File: docs/quick-ingestion-guides/looker/overview.md (hunk 2 above)
Comment (siddiquebagwan) [reply docs/quick-ingestion-guides/looker/overview.md:15]: done

File: docs/quick-ingestion-guides/looker/overview.md (hunk 4 above)
Comment (siddiquebagwan) [reply docs/quick-ingestion-guides/looker/overview.md:33]: done

File: docs/quick-ingestion-guides/looker/configuration.md (hunk 6 above)
Comment (siddiquebagwan) [reply docs/quick-ingestion-guides/looker/configuration.md:115]: done

File: docs/quick-ingestion-guides/looker/setup.md (hunk 5 above)
Comment (siddiquebagwan) [reply docs/quick-ingestion-guides/looker/setup.md:36]: done

File: docs/quick-ingestion-guides/looker/setup.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+2. **Admin Panel:** Navigate to `Admin Panel` on looker home page.
+
+      <p align="center">
+   <img width="75%" alt="Looker home page" src="http://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-home-page.png"/>
+      </p>
+
+3. **Roles Panel:** Search for `Roles` on `Admin Panel` and click `Roles` to open `Roles Panel`.
+
+      <p align="center">
+   <img width="75%" alt="Looker roles search" src="http://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/looker/looker-roles-search.png"/>
```
Comment (yoonhyejin) [inline docs/quick-ingestion-guides/looker/setup.md:23]: This image looks too big - maybe we can modify the width

File: docs/quick-ingestion-guides/looker/overview.md (hunk 2 above)
Comment (yoonhyejin) [reply docs/quick-ingestion-guides/looker/overview.md]: I think the line height here is still too big (see the difference between here and Advanced Guides and Reference)
image

**Review by yoonhyejin — APPROVED** [review]

**Pull request merged**